### PR TITLE
Add information to help you copy

### LINKED ISSUE #15: VVV/E Add copy to 'Information to help you' for seAp
##  Description

We need to add the bite size information we have written based on the user research we have done to the site. 

Link to the master copy: https://docs.google.com/spreadsheets/d/1aFDhogqKhC2AogDuwdOVTlOytb3NxbsYjOaka43ssbw/edit?usp=sharing

## Plan

- [x] Add copy 

## Acceptance Criteria

- [x] Copy matched the copy in the copy spreadsheet 
- [x] All information is selectable from the 'Information to help you' page
- [x] The order of the information is the same as in the Google Spreadsheet

### PR DESCRIPTION
**Pull request by AlinChindea**
Fixes #15 

## Description

- Added copy

- [x] Rose to answer a few clarification questions regarding the copy material before this PR can be merged

@neontribe/ca-seap

### CODE REVIEW

File: src/Copy/BrandCopy.elm [hunk 1]
```diff
@@ -365,76 +365,86 @@ brandCopy key =
                 ]
 
         InfoOneName ->
-            CopyText "✨"
+            CopyText "What happens when you contact us?"
 
         InfoOneSlug ->
-            CopyText "✨"
+            CopyText ""
```
Comment (katjam) [inline src/Copy/BrandCopy.elm]: The slugs are missing.

**Review by katjam — CHANGES_REQUESTED** [review]
I think the icons is part of another ticket. Probably should put the emoji placeholder back for them.

The slugs need to be written in. I think @AlinChindea fine for you to make some sensible ones and get @roseb04 to approve / check.

**Comment by AlinChindea** [comment]
@roseb04 I've added the slugs - the part that comes at the very end of a URL, and refers to a specific page or post - as per @katjam 's request. Can you check if these are fine?

**Comment by rosejbon** [comment]
@AlinChindea those look good to me, good to merge

**Comment by AlinChindea** [comment]
@roseb04 great, I've added those to the copy file.

@katjam do you need to re-approve?

**Review by katjam — APPROVED** [review]

**Pull request merged**